packages/test-app/ember-cli-build.js

```diff
@@ -34,19 +34,26 @@ module.exports = function (defaults) {
       // GitHub Pages uses this filename to serve 404s
       emptyFile: '404.html',
     },
+
+    babel: {
+      plugins: [
+        require.resolve(
+          'ember-concurrency/babel-plugin-transform-ember-concurrency-async-tasks.js'
```

Comment:
That's a long plugin name!

Thoughts on?:

"ember-concurrency/babel"?

It'd require a a prefix to use without require.resolve, like:

module:ember-concurrency/babel

Reply:
Ah yeah, I meant to rename this. Will DM you about `module:...`, don't know too much about that.

Reply:
I shortened it; seems to work fine with `require.resolve`:

```
    babel: {
      plugins: [
        require.resolve('ember-concurrency/async-arrow-task-transform'),
      ],
    },
```